inventory.yaml.sample

```diff
@@ -0,0 +1,5 @@
+all:
```

Comment:
```suggestion
```
all:
```

:)

Reply:
Seems other yaml files are missing it, so maybe you want to skip this one.

Reply:
I'm also curious about the dependabot auto update, wasn't yet confident enough to set it up, but probably a good idea indeed

Reply:
Would be good if you could send a PR to the template repository for this :)

Reply:
https://github.com/EGI-Foundation/repository-template/pull/7